groovyw.bat

```diff
@@ -112,7 +86,7 @@ set CLASSPATH=%APP_HOME%\gradle\wrapper\gradle-wrapper.jar;%APP_HOME%\gradle\wra
 
 
 @rem Execute Groovy via the Gradle Wrapper
-"%JAVA_EXE%" %DEFAULT_JVM_OPTS% %JAVA_OPTS% %GRADLE_OPTS% "-Dorg.gradle.appname=%APP_BASE_NAME%" -classpath "%CLASSPATH%" org.gradle.wrapper.GroovyWrapperMain config/groovy/util.groovy %CMD_LINE_ARGS%
+"%JAVA_EXE%" %DEFAULT_JVM_OPTS% %JAVA_OPTS% %GRADLE_OPTS% "-Dorg.gradle.appname=%APP_BASE_NAME%" -classpath "%CLASSPATH%" org.gradle.wrapper.GroovyWrapperMain %CMD_LINE_ARGS%
```

Comment:
This is the issue. This line should not have been changed. It works for me when I revert that change.

Reply:
but then that must've already went wrong in https://github.com/MovingBlocks/groovy-wrapper, right?

Reply:
or rather it might be specific to how we use the wrapper in the engine...

Reply:
I believe that it was behaving as intended. When you run `groovyw Test` to test the wrapper in its own repository, it is running `Test.groovy`. The first argument to the wrapper is the groovy script to run, I think.